.github/workflows/rust.yml

```
@@ -40,64 +38,44 @@ jobs:
-      - name: Build with all features
-        run: cargo +${{ env.MSRV }} build --all-features
-
   test:
-    needs: [ lint, msrv ]
+    needs: lint
     name: Rust Tests
     runs-on: ubuntu-latest
     timeout-minutes: 30
+    strategy:
+      matrix:
+        rust-version: [ '1.83', 'stable' ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)